### PR TITLE
Add box to tell owner/superuser certain model details are only visible to them

### PR DESCRIPTION
**Pull request by mike-ferguson**
Adds a box around a model dashboard section that says "The following information is only visible to you".

### CODE REVIEW

**Review by mschrimpf — CHANGES_REQUESTED** [review]
`div.visible-to-owner` reinvents the wheel, use e.g. https://bulma.io/documentation/elements/box/

**Comment by mschrimpf** [comment]
can you please post a screenshot of what the site looks like with this change?

File: benchmarks/templates/benchmarks/model.html [hunk 1]
```diff
@@ -5,61 +5,64 @@
...
+                                    <br>
+                                    <br>
+                                {% endif %}
+                            {% endif %}
+            </div>
+            {%  else %}
+            {% endif %}
 
         <span class='information'>Reference Link:</span>
         {% if model.reference_identifier %}
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/model.html:68]: can we please also move this above the fine-print? I.e. have: 
```
<model name>
<reference link>
```

and then the details?

File: benchmarks/templates/benchmarks/model.html [hunk 2]
```diff
@@ -5,61 +5,64 @@
         <h1 class="title">{{ model.name }}</h1>
         <span class='information'>Model Name:</span> {{ model.name }}<br>
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/model.html:6]: remove, `model.name` is already the title

File: benchmarks/templates/benchmarks/model.html [hunk 3]
```diff
@@ -5,61 +5,64 @@
...
+                                    <a class="log_link" href="http://braintree.mit.edu:8080/job/run_benchmarks/{{ model.submission_id }}/consoleText">View Console Log</a>
+                                    <br>
+                                    <br>
+                                {% endif %}
+                            {% endif %}
+            </div>
+            {%  else %}
+            {% endif %}
 
         <span class='information'>Reference Link:</span>
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/model.html:67]: remove this span, let's just have the link

**Review by mschrimpf — CHANGES_REQUESTED** [review]

**Comment by mike-ferguson** [comment]
Screenshot
Screen Shot 2021-12-15 at 10 40 52 AM
:

**Review by mschrimpf — CHANGES_REQUESTED** [review]

File: benchmarks/templates/benchmarks/model.html [hunk 4]
```diff
@@ -1,158 +1,154 @@
...
-    <section class="individual_model container center">
-        <h1 class="title">{{ model.name }}</h1>
-        <span class='information'>Model Name:</span> {{ model.name }}<br>
-        {% if can_see %}
+<section class="individual_model container center">
+    <h1 class="title">{{ model.name }}</h1>
+    {% if model.reference_identifier %}
+        <a href="{{ model.reference_link }}">{{ model.reference_identifier }}</a>
+    {% else %}
+        <p class="no_reference">No reference paper on file.</p>
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/model.html:9]: delete

File: web/settings.py [hunk 5]
```diff
@@ -45,7 +45,7 @@ def get_secret(secret_name, region_name):
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.getenv("DEBUG", "False") == "True"
 
-ALLOWED_HOSTS = os.getenv("DOMAIN", "localhost:brain-score-web-dev.us-east-2.elasticbeanstalk.com").split(":")
+ALLOWED_HOSTS = [os.getenv("DOMAIN", "localhost:brain-score-web-dev.us-east-2.elasticbeanstalk.com").split(":"), "127.0.0.1"]
```
Comment (mschrimpf) [inline web/settings.py:48]: undo this change (fine to keep locally, but not on prod)

**Review by mschrimpf — COMMENTED** [review]
almost done

File: benchmarks/templates/benchmarks/model.html [hunk 6]
```diff
@@ -1,158 +1,153 @@
...
 {% block content %}
-    <section class="individual_model container center">
-        <h1 class="title">{{ model.name }}</h1>
-        <span class='information'>Model Name:</span> {{ model.name }}<br>
-        {% if can_see %}
+<section class="individual_model container center">
+    <h1 class="title">{{ model.name }}</h1>
+    {% if model.reference_identifier %}
+        <a href="{{ model.reference_link }}">{{ model.reference_identifier }}</a>
+    {% else %}
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/model.html:8]: delete the `else` too

**Review by mschrimpf — APPROVED** [review]

**Pull request merged**